prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/FHIRBundleHelpersTests.kt

```diff
@@ -152,6 +173,24 @@ class FHIRBundleHelpersTests {
         assertThat(receiversOut[0]).isEqualTo("co-phd.full-elr-hl7")
     }
 
+    @Test
+    fun `test adding receivers to bundle without provenance`() {
+        // set up
+        val bundle = FhirTranscoder.decode(validFhirWithoutProvenance)
```

Comment:
If I was doing this, I would probably just reuse the existing file in resources? something like:
```
val fhirBundle = File("src/test/resources/fhirengine/engine/valid_data_no_provenance.fhir").readText()
val messages = FhirTranscoder.getBundles(fhirBundle, actionLogger)
assertThat(messages).isNotEmpty()
val bundle = messages[0]
```

Reply:
Updated